pkgs/development/tools/misc/cc-tool/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub,
+  autoconf, automake, boost, libusb1, libtool, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  name = "cc-tool-${version}";
+  version = "unstable-2020-05-19";
+  
+  src = fetchFromGitHub {
+    owner = "dashesy";
+    repo = "cc-tool";
+    rev = "19e707eafaaddee8b996ad27a9f3e1aafcb900d2";
+    sha256 = "1f78j498fdd36xbci57jkgh25gq14g3b6xmp76imdpar0jkpyljv";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libusb1 boost autoconf libtool automake ];
```

Comment:
`autoconf`, `libtool`, and `automake` should be in `nativeBuildInputs`. But see my comment about `autoreconfHook`.